app/lib/components/term.js

```diff
@@ -41,6 +43,9 @@ export default class Term extends Component {
         if (cols !== this.props.cols || rows !== this.props.rows) {
           props.onResize(cols, rows);
         }
+
+      // bell on by default if not defined in the config
+      this.term.bellAudio_.muted = (props.audibleBell == false);
```

Comment:
there's `audible-bell-sound`. It accepts an empty string to disable the bell.
`'lib-resource:hterm/audio/bell'` is the default sound.